benchmarks/python/model_configs.py

```diff
@@ -0,0 +1,100 @@
+# SPDX-FileCopyrightText: Copyright (c) 2024-present NVIDIA CORPORATION & AFFILIATES.
+# All rights reserved.
+# SPDX-License-Identifier: BSD-3-Clause
+from functools import partial
+
+from transformers import AutoConfig
+
+def llama_hf_cfg(config_str):
+    class Config:
+        def __init__(
+            self, n_head, head_size, n_query_groups, rope_n_elem, batches, seq_length
+        ):
+            self.n_head = n_head
+            self.head_size = head_size
+            self.n_query_groups = n_query_groups
+            self.rope_n_elem = rope_n_elem
+            self.batches = batches
+            self.seq_length = seq_length
+
+    configs = {}
+    configs["llama_2_7b_hf"] = Config(
+        n_head=32,
+        head_size=128,
+        n_query_groups=32,
+        rope_n_elem=128,
+        batches=2,
+        seq_length=4096,
+    )
+    configs["llama_3_8B"] = Config(
+        n_head=32,
+        head_size=128,
+        n_query_groups=8,
+        rope_n_elem=128,
+        batches=2,
+        seq_length=8192,
+    )
+
+    return configs[config_str]
+
+def hf_qwen2_cfg():
+    config = AutoConfig.from_pretrained("Qwen/Qwen2.5-7B-Instruct")
+    config.batch_size = 1
+    config.seq_len = 4096
+    config._attn_implementation = "sdpa"
+    return config
+
+def hf_phi3_cfg():
+    config = AutoConfig.from_pretrained("microsoft/Phi-3.5-mini-instruct")
```

Comment:
The existing `phi3` config seems to have a lot of other fields. I am guessing `AutoConfig` fetches the values for all those, and we are deviating with the default config only for the below variables?

Reply:
The existing phi3 config is a save from the AutoConfig load.

The below values only updates _attn_implementation.
Also, the loaded config doesn't give batch size or seq_len, those just specifies input size.